From daa83aac83d8644c645f5b3c6006391214a0864c Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Tue, 18 Feb 2025 09:44:01 +0000
Subject: [PATCH 1/5] avdevice/drm_vout: force 'meson' not 'vc4'

I keep forgetting to specify meson as the drm_vout device so
hardcode this for Amlogic testing.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 libavdevice/drm_vout.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libavdevice/drm_vout.c b/libavdevice/drm_vout.c
index 6d11e98d75..36f9914319 100644
--- a/libavdevice/drm_vout.c
+++ b/libavdevice/drm_vout.c
@@ -40,7 +40,7 @@
 
 #define TRACE_ALL 0
 
-#define DRM_MODULE "vc4"
+#define DRM_MODULE "meson"
 
 #define ERRSTR strerror(errno)
 
-- 
2.43.0

